spec/models/stock_spec.rb

```diff
@@ -1,13 +1,39 @@
 require 'rails_helper'
 
 RSpec.describe Stock, type: :model do
-  context 'ユーザーがログインしている場合' do
-    it '個数が入力されている場合'
-    it '整数ならば有効'
-    it '1/2のような分数なら有効'
-    it 'それ以外なら無効'
-    it 'ユーザーidがなければ無効'
-    it '原材料idが入力されていれば有効'
-    it '原材料idが入力されていなければ無効'
+  # 以下は全てquantityプロパティのvalidationテスト
+  #   describe, contextの使い方は合っているか?
```

Comment:
convert_specific_formatメソッドのテストについて。
メソッドを呼び出した後で、quantityが整形されてるか確認すればいいかと思います。
下記はざっくりしたコードです。

```suggestion
  describe "#convert_specific_format" do
    context "OK" do
      it "空白は除去される" do
        stock = FactoryBot.build(:stock, quantity: " 1 ")
        stock. convert_specific_format
        expect(stock.quantity.to_s).to eq("1")
      end
    end
  end
```